DIRECTOR_HANDBOOK.md

```diff
@@ -0,0 +1,92 @@
+# Node.js Foundation<br>Board of Directors Handbook
+
+Although the Node.js Foundation is a not-for-profit membership organization rather than a corporation owned by stockholders, the same legal rules and obligations apply to its Board of Directors as is the case with a publicly or privately held stock corporation. Thus, while a member of the Board may have been nominated or even appointed by the Director’s employer as an exercise of a membership right, such a director must be mindful of a Director’s obligations under standing law and the Foundation's Certificate of Incorporation and By-Laws. The purpose of this document is to acquaint new and existing Directors of these obligations.
+
+It is the express policy of the Foundation to require that all its activities be conducted strictly in accordance with federal and state laws. It is extremely important that all members of the Board of Directors be aware of the legal and ethical standards to which you are held and the types of activities to avoid in order to fulfill your duties as directors properly, as well as to avoid liability.
+
+These Governance Guidelines, Code of Conduct and Duties were prepared to provide you with a broad, summary overview of the standards under the law to which you will be held. However, you should note that these governance guidelines and standards provide a general guide only; they are not intended to be a complete and definitive statement of all aspects of the law relating to the duties of Directors. Any specific questions relating to your duties as a Director not addressed in these governance guidelines and standards should be referred to counsel for the Foundation or to your own legal counsel who has responsibility for considering your duties as a Director of the Foundation.
+
+## Board Governance
+
+The Board of Directors represents the interests of the Foundation’s Members. The Board is the ultimate decision-making body of the Foundation except with respect to those matters reserved to the Members. The business and affairs of the Foundation are managed by its Board. Although the Board may delegate authority, the ultimate responsibility for the Foundation resides with the Board, which may not lawfully abdicate its duties.
```

Comment:
I think this should be clarified a bit. While fundamental responsibility for the Foundation as a legal entity falls to the Board, the Foundation was established in such a way that grants the TSC autonomy over the technical aspects of the project. We should not need to completely get into all of the nuances of that here, but we should refine the language a bit to say something along the lines of `The Board is fundamentally responsible for the day-to-day legal, marketing, and financial affairs of the Foundation while responsibility for the technical evolution of Foundation projects is delegated to the Technical Steering Committee.` That's not a great way of putting it either but hopefully the point gets across -- that is, there are simply some aspects of what the Foundation does (specifically the technical direction) that is expressly *not* ultimately the responsibility of the Board.